src/gui/Lv2ViewBase.cpp

```diff
@@ -248,6 +251,32 @@ AutoLilvNode Lv2ViewBase::uri(const char *uriStr)
 }
 
 
+
+
+void Lv2ViewBase::onHelpWindowClosed()
+{
+	m_helpButton->setChecked(true);
+}
+
+
+
+
+HelpWindowEventFilter::HelpWindowEventFilter(Lv2ViewBase* viewBase) :
+	m_viewBase(viewBase) {}
+
+
+
+
+bool HelpWindowEventFilter::eventFilter(QObject* , QEvent* event)
```

Comment:
I was not sure whether I should install the `eventFilter` on a separate object (as here, on `HelpWindowEventFilter`), or directly in `Lv2ViewBase`.